### ISSUE TITLE
Paper Revision 2023.at4ssl-1.6

### ISSUE THREAD

**Issue by ZaidAliMohammed** [acl-org/acl-anthology#2772]
### Anthology ID

2023.at4ssl-1.6

### Type of Change

Revision

### PDF of the Revision or Erratum

[AT4SSL_2023_paper_9.pdf](https://github.com/acl-org/acl-anthology/files/12537297/AT4SSL_2023_paper_9.pdf)

### Brief Description of Changes

Dear all, 

This is Zaid Mohammed, the first author of the paper namely "Towards Accommodating Gerunds within the Sign Language Lexicon", I am requesting a revision for the paper as the published version is different from the version that was actually provided at the AT4SSL workshop, as appears here : https://sites.google.com/tilburguniversity.edu/at4ssl2023/proceedings?authuser=0.

I would really appreciate if you could update the published version with the one that I have attached.

Best Regards,
Zaid Mohammed

**Comment by anthology-assist** [comment]
@ZaidAliMohammed Could you summarize what has changed in the paper?

**Comment by ZaidAliMohammed** [comment]
Dear all,

The paper has been updated in light of the advances in the research, in
which gerund constructions were found to be related to aspect and
aspectual markers in signed languages, i.e. there was a need to mention
that a deeper research on aspectual marking is required and update the
paper accordingly.

Best regards,
Zaid Mohammed
